server/server_static.go

```diff
@@ -0,0 +1,101 @@
+package server
+
+import (
+	"embed"
+	"errors"
+	"io/fs"
+	"mime"
+	"net/http"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	//go:embed static/*
+	static         embed.FS //nolint
+	staticBasePath = "static"
+	IndexHtmlFile  = staticBasePath + "/index.html"
```

Comment:
Is there a reason this is public?

Reply:
Oops, well spotted.
Not by Intend and can be private.

```suggestion
	indexHtmlFile  = staticBasePath + "/index.html"
```